yaksh/scripts/cli.py

```diff
@@ -21,61 +18,68 @@
 settings.configure()
 django.setup()
 
+
 def main():
-    #Parse command-line to obtain the arguments and/or options
+    # Parse command-line to obtain the arguments and/or options
     # create top-level parser object
     parser = argparse.ArgumentParser(prog="yaksh")
     subparser = parser.add_subparsers(dest="subcommand")
 
     # create parser for the "create_demo" subcommand
-    create_demo_parser = subparser.add_parser("create_demo",
-                                        help="Create a new demo Django project")
-    create_demo_parser.add_argument("project_name", type=str,
-                                        default="yaksh_demo",
-                                        nargs="?",
-                                        help="name of demo Django project")
-    create_demo_parser.add_argument("-p", "--path", type=str,
-                                        help="path of demo Django project")
-    
+    create_demo_parser = subparser.add_parser(
+        "create_demo",
+        help="Create a new demo Django project"
+    )
+    create_demo_parser.add_argument(
+        "path", type=str,
+        default=path.join(CUR_DIR, "yaksh_demo"),
+        nargs="?",
+        help="Path to the demo Django project (defaults to 'yaksh_demo')"
+    )
+
     # create parser for the "run_demo" subcommand
-    run_demo_parser = subparser.add_parser("run_demo",
-                        help="Initialise django server and run the demo project")
+    run_parser = subparser.add_parser(
+        "run",
+        help="Initialise django server and run the demo project"
+    )
+    run_parser.add_argument(
+        "path", type=str, nargs=1,
```

Comment:
shouldn't this have an additional argument for ports? In case if something has already occupied the default port.